src/components/ReportActionItem/MoneyRequestReportPreview/EmptyRequestReport.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {View} from 'react-native';
+import Button from '@components/Button';
+import * as Expensicons from '@components/Icon/Expensicons';
+import ImageSVG from '@components/ImageSVG';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+
+function EmptyRequestReport() {
```

Comment:
NAB but I wanted to propose a different name, reasons:
1. `Report` has some special meaning and just plain word report means usually a full-screen report view, or report-data. 
2. we shouldn't use the name `RequestReport` because there is no such thing. There is `MoneyRequestReport`, but if we cut out the "money" part then it gets confusing

My suggestions:
 - `EmptyMoneyRequestReportPreview`
 - `MoneyRequestReportPreviewSkeleton`
 - `EmptyReportPreview`

or anything similar

Reply:
I'm already taking care of the changes